pkg/frontend/mysql_cmd_executor.go

```diff
@@ -3327,11 +3330,12 @@ func (mce *MysqlCmdExecutor) processLoadLocal(ctx context.Context, param *tree.E
 	if length == 0 {
 		return
 	}
+	quickWrite := false
```

Comment:
quickWrite is confusing. What does it works?

Reply:
added some comment in code